app/src/main/java/org/fossasia/openevent/general/search/SearchTimeActivity.kt

```diff
@@ -36,11 +36,12 @@ class SearchTimeActivity : AppCompatActivity() {
             calendar.add(Calendar.DATE, 1)
             searchTimeViewModel.saveNextDate(simpleDateFormat.format(calendar.time))
 
-            val intent = Intent(this, MainActivity::class.java).setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP)
-            val bundle = Bundle()
-            bundle.putBoolean(TO_SEARCH, true)
-            intent.putExtras(bundle)
-            startActivity(intent)
+            redirectToSearch()
+        }
+
+        anytimeTextView.setOnClickListener {
+            searchTimeViewModel.saveDate("Anytime")
```

Comment:
This can be used as a constant so that you can refer it from other places as well

Reply:
> This can be used as a constant so that you can refer it from other places as well

@iamareebjamal Then how to use it as constant?

Reply:
`const name = "Areeb"`

I guess this doesn't work? If so, then let it be

Reply:
> `const val name = "Areeb"`
> 
> I guess this doesn't work? If so, then let it be

It's working. Updated.